arduino/arduino-ide
# Translating Arduino-IDE using Theia's nls API

Hi team,

this PR adds translation keys for all user-facing strings inside of the Arduino-IDE using the Theia localization (nls) API. 

Be aware that this does not add actual translations for the app. However, some strings are nevertheless translated, as they are contained by the vscode language pack.

### How to test

0. Start the app and observe that all user-facing strings are still correct (in English)
1. Download a language pack of your choice from the vscode marketplace/ovsx registry.
2. Unzip it and place it into the `plugins` folder
3. Start the application and run the `Configure Display Language` command and change to your new language
4. Open the `File` menu.
5. Observe that some menu entries have been changed.
